tests/src/test/scala/whisk/core/controller/test/TriggersApiTests.scala

```diff
@@ -316,17 +317,20 @@ class TriggersApiTests extends ControllerTestCommon with WhiskTriggersApi {
     val content = JsObject("xxx" -> "yyy".toJson)
     put(entityStore, trigger)
     Post(s"$collectionPath/${trigger.name}", content) ~> Route.seal(routes(creds)) ~> check {
-      status should be(OK)
+      status should be(Accepted)
       val response = responseAs[JsObject]
       val JsString(id) = response.fields("activationId")
       val activationId = ActivationId(id)
       response.fields("activationId") should not be None
 
       val activationDoc = DocId(WhiskEntity.qualifiedName(namespace, activationId))
-      val activation = get(activationStore, activationDoc, WhiskActivation, garbageCollect = false)
-      del(activationStore, DocId(WhiskEntity.qualifiedName(namespace, activationId)), WhiskActivation)
-      activation.end should be(Instant.EPOCH)
-      activation.response.result should be(Some(content))
+      whisk.utils.retry({
```

Comment:
This could have used the withActivation helper I think. And below.

Reply:
Isn't that more geared towards system level tests? It uses "real" communication if i'm not mistaken (using Wsk or WskRest) vs. akka-http "internal" tests here.

Reply:
i looked at some of those helpers and it seemed to me that they depended on having a `Wsk` object to drive the requests, which was not the case for these tests